### PR TITLE
paramteter for preselect tenant

### PR DESCRIPTION
**Pull request by ibsz**
if "preselect_tenant" set, it will contributed to login request

### CODE REVIEW

File: pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 public class PartnerNetOAuth2AuthorizationRequestResolver implements OAuth2AuthorizationRequestResolver
 {
     public static final String ACR_PARAM = "acr";
+    public static final String PRESELECT_TENANT_PARAM = "preselect_tenant";
     private final OAuth2AuthorizationRequestResolver defaultAuthorizationRequestResolver;
 
     public static UriComponentsBuilder requestNistAuthenticationLevels(UriComponentsBuilder uri,
```
Comment (furti) [inline pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java:32]: Please add a "requestPreselectedTenant" method the same way as the requestNistAuthenticationLevels method.

**Review by furti — CHANGES_REQUESTED** [review]

File: pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java (hunk 1 above)
Comment (ibsz) [reply pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java:32]: method added

File: pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java [hunk 2]
```diff
@@ -39,6 +40,16 @@ public static UriComponentsBuilder requestNistAuthenticationLevels(UriComponents
         return uri;
     }
 
+    public static UriComponentsBuilder requestPreselectTenant(UriComponentsBuilder uri,
+    String preseletedTenant)
```
Comment (furti) [inline pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java:44]: Sorry for the late reply. There is a typo in the paramter name. It should be presele **c** tedTenant

**Comment by thred** [comment]
I think this is outdated

**Pull request closed without merging**